README.md

```diff
@@ -1,2 +1,107 @@
-# eslint-udemy
-ESLint configs and plugins used by Udemy.
+Udemy ESLint Packages
+=====================
+
+This repository has [*ESLint*](https://www.github.com/eslint/eslint) related packages used by various Udemy projects. 
```

Comment:
Why do your link bodies have `*` in them?

Reply:
I italic-case keywords (e.g. library names) the *first* time they appear in a doc.